BookPlayer/Coordinators/LibraryListCoordinator.swift

```diff
@@ -190,12 +190,23 @@ class LibraryListCoordinator: ItemListCoordinator {
     }
   }
 
-  override func interactiveDidFinish(vc: UIViewController) {
+  func navigationController(_ navigationController: UINavigationController, didShow viewController: UIViewController, animated: Bool) {
```

Comment:
Some context here: the idea is to move away from the detach / attach pattern of the coordinators, and have them be kept in memory by the view models, so when the flow from the coordinator is dismissed, it's automatically deallocated, and we would no longer need to have these checks